weather_app.py

```diff
@@ -1,14 +1,17 @@
+
 import requests
 from config import openweather_api_key as api_key
 
-def get_weather(city_name):
-    base_url = "http://api.openweathermap.org/data/2.5/weather"
+def get_weather(city_name, forecast=False):
+    base_url = "http://api.openweathermap.org/data/2.5/weather" if not forecast else "http://api.openweathermap.org/data/2.5/forecast"
     params = {
         'q': city_name,
         'appid': api_key,
         'units': 'metric'
     }
     response = requests.get(base_url, params=params)
+    if response.status_code != 200:
```

Comment:
you could also add a `response.raise_for_status()` but then you do need to catch the exception where this function gets called